onelogin_rules/onelogin_brute_force_by_ip.yml

```diff
@@ -16,6 +16,12 @@ Threshold: 10
 DedupPeriodMinutes: 10
 Reference: https://developers.onelogin.com/api-docs/1/events/event-resource
 Runbook: Analyze the IP they came from, and other actions taken before/after. Check if a user from this ip eventually authenticated successfully.
+SummaryAttributes:
+  - event_type_id
+  - actor_user_name
+  - user_name
+  - actor_user_id
+  - user_id
```

Comment:
would `ipaddr` or `p_any_ipaddresses` be useful?

Reply:
Don't see any reference to those in the tests but yeah, I think it logically makes sense to include `p_any_ip_addresses`